storage/src/lib.rs

```diff
@@ -146,6 +146,13 @@ pub enum CheckerError {
         expected_free_list: AreaIndex,
     },
 
+    /// Found leaked areas
+    #[error("Found leaked areas: {leaked:?}")]
```

Comment:
This might be a massive string. Suggest using `leaked.get(0..SOME_SANE_MAXIMUM)` to avoid that